### ISSUE TITLE
Explicitly Disallow Running Water

### ISSUE THREAD

**Issue by JacksonBurns** [JacksonBurns/fastsolv#40]
Water is not included in the training data for `fastsolv`, so while the gradients might be accurate the absolute predictions will likely very off.

It would be good to disallow running water, especially on the public-facing website, and on the `fastsolv` Python package as well.

One way to do this would be to raise an `Exception` of some kind.

Of course, this may change in the future was we look to extend `fastsolv`.

**Comment by Ruruthia** [comment]
Hey, out of curiosity - why water for excluded from the training data? I saw this [closed PR](https://github.com/JacksonBurns/fastsolv/pull/11) where you tried to add separate featurization for water and it didn't perform well. Did you also observe bad performance when adding water without featurizing it separately, treating like any other solvent?

**Comment by JacksonBurns** [comment]
@Ruruthia when we added water to the model, the results for water ranged from "about as good as the literature" to worse than the literature. Under some approaches, including water also made our performance on organics (which is better than the literature alternatives) worse as well. So for practical performance reasons we left it out.

In a larger sense, we believe aqueous solubility prediction should be treated as a different task. Drug discovery often only considers it in a very narrow range, and only at one temperature; many industrial processes use organic solvents rather than water; literature contains many _very_ low reported aqueous solubility values because of research areas like environmental chemistry and pesticides; etc.

There's additional practical limitations, like how our model doesn't recognize temperatures above the boiling points for common organics, though water would still be liquid at those temperatures.
